README.md

```diff
@@ -35,27 +33,22 @@ Spartacus provides core storefront features such as:
 
 See the [Release documentation](https://sap.github.io/cloud-commerce-spartacus-storefront-docs/release-information/) for more information.
 
-
-
 # Requirements
 
```

Comment:
These are the requirements for 2.0, maybe specify that? and point out that 1.x are different? also where do they find 1.x requirements.

Reply:
Good point! The 1.x requirements have been added back in.